test/train-sets/ref/help.stdout

```diff
@@ -443,7 +443,7 @@ Weight Options:
                                             necessary)
     --cb_type arg                           Contextual bandit method to use (type: str, default: dr, choices
                                             {dm, dr, ips, mtr, sm}, keep)
-    --eval                                  Evaluate a policy rather than optimizing (type: bool)
```

Comment:
eval should not be keep

Reply:
linked issue #4175, seems unrelated to this PR